packages/paste-website/src/pages/page-templates/wizard/index.mdx

```diff
@@ -43,7 +47,189 @@ export const getStaticProps = async () => {
+If your app doesn’t allow loading a page without the navigation frame, [collapse the Sidebar](/components/sidebar#compact-push-sidebar) and follow the [default Wizard template](#default-template).
+
+#### Page header
+
+At the top of the page, add the Task [Heading](/components/heading) and if needed, [Paragraph](/components/paragraph) text under the heading to explain the step.
+
+#### Body
+
+Follow the [default Wizard template](#default-template).
+
+#### Actions footer
+
+Follow the [default Wizard template](#default-template), but don’t include a close button in the footer.
+
+### Using Progress Steps
+
+Only use [Progress Steps](/components/progress-steps) when necessary. Some scenarios include when:
+
+- A customer needs to return to previous steps, especially across multiple sessions.
+- A goal takes a long time to finish.
+- A customer is trying a feature out or going through more of an exploratory flow. Progress Steps can help them understand if they want to complete the steps.
+
+If the customer’s journey through a Wizard is relatively linear, continuous, and short, don’t use Progress Steps.
+
+When in doubt, start by not using Progress Steps, and add it in once it’s clear the experience suffers without it.
+
+<Callout variant='neutral' marginY="space70">
+  <CalloutHeading as="h2">Research insight</CalloutHeading>
+  <CalloutText><Anchor href="https://designnotes.blog.gov.uk/2014/07/07/do-less-problems-as-shared-spaces/">Research from GOV.UK</Anchor> showed that their users were actually getting confused and intimidated when seeing their progress indicator. They found that completion rates, time-to-completion, and total amount of online applications were unaffected after removing their progress bar from their services.</CalloutText>
+</Callout>
+
+Match Progress Steps labels to task headings for each page. If you need to use Progress Steps and there’s forking logic in the flow, hide Progress Steps until there’s a single path forward, or make the step labels generic enough to cover all variants of a task.
+
+#### With the default template
+
+Place Progress Steps above the Wizard title.
+
+<StoryPreview
+  title="Wizard page template - default header with progress steps"
+  storyID="website-pagetemplateexamples--wizard-default-header-with-progress-steps-example"
+  code={WizardDefaultHeaderWithProgressStepsExample}
+  height="420px"
+/>
+
+#### With the full-page template
+
+Center Progress Steps in the Topbar.
+
+<StoryPreview
+  title="Wizard page template - full page header with progress steps"
+  storyID="website-pagetemplateexamples--wizard-full-page-header-with-progress-steps-example"
+  code={WizardFullPageHeaderWithProgressStepsExample}
+  height="150px"
+/>
+
+## Starter kits
+
+#### CodeSandbox
+
+Coming soon
+
+#### Figma
+
+Add the “[Paste Patterns & Templates](https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns-%26-Templates?type=design&node-id=367-27339&mode=design&t=dC1B5k0lVIB0JIuX-0)” library to your file.
```

Comment:
chore: Let's just remove this since the coded examples are prob enough, and Figma's the same for every template.